net/lighttpd/Makefile

```diff
@@ -77,15 +76,39 @@ else
 	--without-openssl
 endif
 
-ifneq ($(SDK)$(CONFIG_PACKAGE_lighttpd-mod-mysql-vhost),)
+ifneq (,$(CONFIG_PACKAGE_lighttpd-mod-authn-gssapi))
```

Comment:
Why do you change the style in the ifnew clauses? starting with comma? Looks strange and is not usual here.

And is there a reason why you drop the SDK condition from the clause?